editor/src/clj/editor/scene.clj

```diff
@@ -600,6 +600,22 @@
             union-aabb
             (:children scene))))
 
+(declare active-scene-view)
+(declare frame-selection)
```

Comment:
We try to avoid using `declare` if possible, since it might be a symptom of cyclic dependencies in code. Instead, we could move the definitions of the function here, or move the use of these functions lower.

Reply:
I've removed it, thx